Gemfile

```diff
@@ -1,3 +1,34 @@
-source "http://rubygems.org"
-
-gemspec
+                   ,    ,    .
+               , ~@  `@ @~  `@  ,
+             ~@ @ZXZ%%X&ZX%Z%XZ@`,
+          ;@ %  @.~@,-.&&,-.@~ @ @H @~
+          ,@X  ~  @(   )(   )@"  ~@X
+          H  @     )   ()   (      ;@H@. ,
+      `@X ,   `   '-=o=-'=o=-'         %@
+   `@ %  @                            ,@ X@~
+   ~ X@   "                            "  %  ,
+  ;@H                      ,-.             H@.
+    %@~            .,.    (/)_)          `@X
+    H `          ,*@@@*.  d " b          ,@%@~
+    %@~           &&&-b    \ /           ~@%
+    X@.           && /: ,-/[x]\-.        ' X@
+  ~@H              &!! /  \|M|/  \         H `
+   'X@           /](  )[\ /|M|\~| |        X@:
+    H           | ( ~~ ) !\| |/ | |      `@%
+    H@.         `='8  [`=' |-|  | |      ~ H
+  ,@X            \\(@*)//  |-|  |/         H@~
+    %@~         / (*@@*) \_| |__|        `@X
+    H `        /   (*@)   \ |  |         ,@%@~
+    X@        /  ,~ ;: ~`  \|  |           H
+  `@% '      /     :  ;     \  |         ~@% ,
+    H       /~       ;       \ |           X@.
+    X@.    /.,   ~@~    ~@~   \|           H
+    H      /  '"*.,,*"'*,.,*'"\|         `@H
+   @X@~   /                    \          X@
+  ~ %@,  /                      \        ,@H ~
+    H   /                        \         H
+    H@.@~                       ~@\        %@,
+  ,@X `'"*'*,  ~@~     ~@~  ,.*'"*"      ~@X
+    H@~      '"*,.*"'"*.,*"'               H@.
+    H><>gpyy<><><><><><><><><><><><><><><><H
+    
```

Comment:
Congratulations on your last day of single life and first day of married life!!!